src/dos/drive_local.cpp

```diff
@@ -218,6 +218,12 @@ bool localDrive::GetSystemFilename(char *sysName, char const * const dosName) {
 
 // Attempt to delete the file name from our local drive mount
 bool localDrive::FileUnlink(char * name) {
+	if (!FileExists(name)) {
+		DEBUG_LOG_MSG("FS: Skipping removal of %s because it doesn't exist",
+		              name);
```

Comment:
https://dos4gw.org/41h_65_Delete_File_UNLINK

Although I haven't tested it yet, I will try returning false with a FILE_NOT_FOUND code set, which is more correct.